src/main/resources/rhsm-conduit.properties

```diff
@@ -1,3 +1,5 @@
 # Place application related defaults here.  This file is loaded via the @PropertySource annotation on the
 # ApplicationConfiguration class.  Prefix properties in this file appropriately (e.g. "rhsm-conduit") so the
 # classes annotated with @ConfigurationProperties will ingest them.
+server.servlet.context-path=${PATH_PREFIX:/r/insights/platform}
```

Comment:
Can we default this to "" so that by default during development the app's root URL is /rhsm-conduit ?

When we deploy to openshift we can update this via the config mapping to be: /r/insights/platform

I think this would make curling the API during development easier.